### PR TITLE
[PIL-1025][AV] - Copy Changes (Dashboard & Amend Subscription screens)

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -692,38 +692,38 @@ useContactPrimary.email = Email
...
 contactNameCompliance.heading.caption = Contact details
-contactNameCompliance.title = What is the name of the person or team we should contact about compliance with Pillar 2 top-up taxes?
-contactNameCompliance.heading = What is the name of the person or team we should contact about compliance with Pillar 2 top-up taxes?
+contactNameCompliance.title = What is the name of the person or team we should contact about compliance for Pillar 2 top-up taxes?
+contactNameCompliance.heading = What is the name of the person or team we should contact about compliance for Pillar 2 top-up taxes?
 contactNameCompliance.hint =  For example, ‘Tax team’ or ‘Ashley Smith’.
-contactNameCompliance.error.required = Enter name of the person of team we should contact
-contactNameCompliance.error.required = Enter name of the person of team we should contact
-contactNameCompliance.error.length = The name of the contact person or team should be 160 characters or less
+contactNameCompliance.error.required = Enter name of the person of team we should contact about compliance for Pillar 2 top-up taxes
```
Comment (jassalrichy) [inline conf/messages.en:698]: contactNameCompliance.error.required value does not match prototype.

File: conf/messages.en [hunk 2]
```diff
@@ -692,38 +692,38 @@ useContactPrimary.email = Email
...
+contactEmailAddress.error.required = Enter the email address for {0}
+contactEmailAddress.error.length = Email address must be 132 characters or less
 contactEmailAddress.error.format = Enter an email address in the correct format, like name@example.com
 contactEmailAddress.change.hidden = the first contact email address
 
 contactByTelephone.heading.caption = Contact details
 contactByTelephone.title = Can we contact by telephone?
 contactByTelephone.heading = Can we contact {0} by telephone?
+contactByTelephone.hint = We will only use this to contact you about Pillar 2 top-up taxes.
 contactByTelephone.checkYourAnswersLabel = Can we contact by telephone?
```
Comment (jassalrichy) [inline conf/messages.en:716]: Should contactByTelephone.checkYourAnswersLabel also include the name param ?

File: conf/messages.en [hunk 3]
```diff
@@ -692,38 +692,38 @@ useContactPrimary.email = Email
...
-contactNameCompliance.error.length = The name of the contact person or team should be 160 characters or less
+contactNameCompliance.error.required = Enter name of the person of team we should contact about compliance for Pillar 2 top-up taxes
+contactNameCompliance.error.length = Name of the contact person or team must be 160 characters or less
 contactNameCompliance.error.invalid = The name of the contact person or team must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, commas, forward slashes, full stops, hyphens, round brackets and spaces
 
 contactEmailAddress.heading.caption = Contact details
 contactEmailAddress.title = What is the email address?
 contactEmailAddress.heading = What is the email address for {0}?
 contactEmailAddress.hint = We will only use this to contact you about Pillar 2 top-up taxes.
 contactEmailAddress.checkYourAnswersLabel = Email address
```
Comment (jassalrichy) [inline conf/messages.en:706]: Should contactEmailAddress.checkYourAnswersLabel also contain the name param ?

File: conf/messages.en (hunk 1 above)
Comment (aashik-varahan) [reply conf/messages.en]: Updated

File: conf/messages.en (hunk 2 above)
Comment (aashik-varahan) [reply conf/messages.en:716]: No, as per prototype contactByTelephone.checkYourAnswersLabel doesn't include name param

File: conf/messages.en (hunk 3 above)
Comment (aashik-varahan) [reply conf/messages.en:706]: No, as per prototype contactEmailAddress.checkYourAnswersLabel doesn't include name param

**Review by jassalrichy — DISMISSED** [review]

**Pull request closed without merging**